src/Coin.cpp

```diff
@@ -28,6 +29,9 @@ using namespace TW;
 
 std::string TW::loadAddress(TWCoinType coin, const Data& data) {
     switch (coin) {
+    case TWCoinTypeOntology:
```

Comment:
Please order alphabetically.

Reply:
This is a strict requirement...

Reply:
Sarcasm is not appropriate for code review comments. If you don't agree with may comments please say so in plain language.

Reply:
I'm sorry, let's continue to cooperate.

Reply:
It is not a strict requirement, just a suggestion.

Reply:
OK, I'm sorry, I will adopt this suggestion in my code.